### DISCUSSION TITLE
Getting PRs for #57

### DISCUSSION THREAD

**Discussion by XotEmBotZ** [Jeffser/Alpaca#73] [General]
I am not very familiar with the gtk framework but I am very comfortable with Python. For past few months I have been trying to learn Gtk but never got motivation. Since I use this app, I will be very motivated to work and it will let me learn also. Do you mind if I get on these issues and let me know which issues are already in development. 

(At first I would like to do python only tasks to get familiarized)

**Comment by Jeffser** [comment]
Well if you want to you can make the logging of the app, I think imdev gave up when I changed like half of the code when they were doing that. Just stuff like saying "[pulling] llama3" "[message-send] ..."
Stuff like that
https://github.com/Jeffser/Alpaca/issues/66
https://github.com/Jeffser/Alpaca/pull/37

**Reply by XotEmBotZ** [reply]
Will proceed with that. Though I am facing some problems while running the code. 
when ever I run python -m src.main it throws error `gi.repository.GLib.GError: g-resource-error-quark: The resource at “/com/jeffser/Alpaca/window.ui” does not exist (0)`
and if I fix it form `@Gtk.Template(resource_path='/com/jeffser/Alpaca/window.ui')` to  `@Gtk.Template(filename='src/window.ui')` it just executes without opening and exits. What could be the problem?

It would be better to give instructions on how to run the project locally. (Without GNOME builder)

**Reply by Jeffser** [reply]
if you want to help use GNOME Builder

**Reply by XotEmBotZ** [reply]
> if you want to help use GNOME Builder

It is too slow and laggy for my system, IDK why but it is. Isn't there any other way of running the application?

**Reply by loulou64490** [reply]
> > if you want to help use GNOME Builder
> 
> It is too slow and laggy for my system, IDK why but it is. Isn't there any other way of running the application?

Do you try your system package ? Or on the contrary the flatpak version ?

**Reply by XotEmBotZ** [reply]
> > > if you want to help use GNOME Builder
> > 
> > 
> > It is too slow and laggy for my system, IDK why but it is. Isn't there any other way of running the application?
> 
> Do you try your system package ? Or on the contrary the flatpak version ?

system package. Though I found a way of running the application first by compiling it to flatpack then running the application. I guess this should work for me now. If you have some other options to make it better please let me know.

**Reply by XotEmBotZ** [reply]
Something unexpected came up. I guess I wont be able to make it. I don't to leave that "someday" I will push the changes. I am withdrawing from this now. Will help when I get time.

**Reply by Jeffser** [reply]
don't worry, good luck with that
